core/src/test/java/org/elasticsearch/search/aggregations/metrics/InternalStatsTests.java

```diff
@@ -32,10 +32,10 @@
     @Override
     protected InternalStats createTestInstance(String name, List<PipelineAggregator> pipelineAggregators,
                                                Map<String, Object> metaData) {
-        long count = randomIntBetween(0, 50);
-        double min = randomDoubleBetween(-1000, 1000, false);
-        double max = randomDoubleBetween(-1000, 1000, false);
-        double sum = randomDoubleBetween(-1000, 1000, false);
+        long count = frequently() ? Integer.MAX_VALUE : 0;
```

Comment:
randomPositiveLong() won't work though I think because then we run the risk of overflowing long when summing those values. Interger.MAX_VALUE should work I think.